client/src/main/scala/caliban/client/CalibanClientError.scala

```diff
@@ -38,7 +38,7 @@ object CalibanClientError {
             s" at path ${p.map {
               case Left(value)  => s"/$value"
               case Right(value) => s"[$value]"
-            }.mkString("")}"
+            }.mkString("")}${e.extensions.fold("")(ext => s" Extensions: $ext")}"
```

Comment:
We need to be a bit careful here. If the server includes traces in the response, this could end up being a _very_ large error message.

Reply:
Sure. 
What do you suggest? 
Some characters limit? Or something else?

My motivation for this is the above ☝🏼  error message. 

I consider it useful to be shown.

Reply:
I think character limit makes sense, although we'll need to preserve the current behaviour so the character limit should be applied for extensions only, with a message as `< error message truncated due to size >` at the end (or something along these lines)

Also, I would suggest changing the method to use a `StringBuilder` instead of string interpolation, as it's more performant and makes things a bit more readable. With a StringBuilder you can keep checking its size as well as you append more to it

Reply:
We could potentially have another method as well which doesn't truncate the error message

Reply:
>We could potentially have another method as well which doesn't truncate the error message

Yes, sounds like a better idea. So existing behaviour doesn't change
I have updated the PR